community/hugo/APKBUILD

```diff
@@ -1,40 +1,40 @@
 # Contributor: Thomas Boerger <thomas@webhippie.de>
 # Maintainer: Thomas Boerger <thomas@webhippie.de>
 pkgname=hugo
-pkgver=0.30.2
+pkgver=0.39
 pkgrel=0
 pkgdesc="A Fast and Flexible Static Site Generator built with love in GoLang"
 url="http://gohugo.io/"
 arch="all"
 license="Apache-2.0"
 depends=""
-makedepends="go govendor"
+makedepends="go dep"
 install=""
 source="${pkgname}-${pkgver}.tar.gz::https://github.com/gohugoio/hugo/archive/v$pkgver.tar.gz"
 builddir="$srcdir/src/github.com/gohugoio/$pkgname"
 
 prepare() {
 	mkdir -p ${builddir%/*}
-	mv "$srcdir"/$pkgname-$pkgver "$builddir"/ || return 1
+	mv "$srcdir"/$pkgname-$pkgver "$builddir"/
 	default_prepare
 }
 
 build() {
 	cd "$builddir"
 	export GOPATH="$srcdir"
-	govendor sync -v || return 1
-	go build -v -o bin/$pkgname || return 1
+	dep ensure -v
+	go build -v -o bin/$pkgname
 }
 
 check() {
 	rm -f $builddir/releaser/git_test.go
 	cd "$builddir"
 	export GOPATH="$srcdir"
-	govendor test +local || return 1
+	go test
```

Comment:
>>> hugo*: check
?       github.com/gohugoio/hugo        [no test files]

Could this be the issue? can you check what going on here?

Reply:
Ah yeah, it doesn't find tests in the root package. But this exits with 0.

Reply:
I have enabled the tests recursively now.